about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <title>Capstone Project x01</title>
+</head>
+<body>
+        <div class="container">
+          <div class="loggings desktop">
+            <a href="#"><img src="./assets/desktop/facebook.png" alt="facebook"></a>
+            <a href="#"><img src="./assets/desktop/twitter.png" alt="twitter"></a>
+            <a href="#"><p>Logout</p></a>
+            <a href="#"><p>Mypage</p></a>
+            <a href="#"><p>Korean</p></a>
+          </div>
+          <header>
+            <img class="hambuger" src="./assets/mobile//hamburger.png" alt="harmbuger icon"/>
+            <nav class="navigator toggle-ON">
+              <button class="exit-btn">X</button>
+              <a href="index.html"><img class="desktop" src="./assets/mobile/logo.svg" alt="logo"></a>
+              <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="desktop pdc"><a href="#">PD Campaign</a></li>
+              </ul>
+              <div>
```

Comment:
- [ ] as the previous reviewer requested kindly follow the Figma design colors patterns please change the background color of the top bar to match the template and kindly fix the font colors , same thing as well for the navbar 

your design 
![image](https://user-images.githubusercontent.com/89282221/190210481-28992d7d-bd98-4c62-9ecf-916a689b8921.png)

Required design 
![image](https://user-images.githubusercontent.com/89282221/190210618-62e0ed2d-1e2e-40f1-9dd5-06ac8fad5d33.png)

Reply:
Alright. adjusted.